SU-SWS/saa_victoria_subtheme
# ADAPT-515 Updated user login page to use ADAPT branding and colors

# READY FOR REVIEW

# Summary
- Updated user login page to use ADAPT branding and colors

# Review By (Date)
- asap

# Urgency
- medium

# Steps to Test

1. Pull this branch to local
2. Clear site cache
3. Navigate to `user/login`. Depending on whether or not you have sso/saml configured, the login screen should appear like so:
![image](https://user-images.githubusercontent.com/34019925/88191559-6c6e9f00-cbf0-11ea-8260-11e3837cc2d0.png)
**Note:** The blue task bar is inherited from core. The `Page Content` label will be removed and a `Back to site` button will be added in the next core release.


# Associated Issues and/or People
-  [ADAPT-515](https://stanfordits.atlassian.net/browse/ADAPT-515)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
